pages/common/autossh.md

```diff
@@ -0,0 +1,21 @@
+# autossh
+
+> Runs, monitors and restarts SSH connections.
+> Auto-reconnects to keep port forwarding tunnels up. Accepts all ssh flags.
+
+- Open an SSH connection with port forward monitoring and forward a local port to a remote one.
+
+`autossh -M {{monitor_port}} {{ssh_command}}`
+`autossh -M {{monitor_port}} -L {{local_port}}:localhost:{{remote_port}} {{user}}@{{host}}`
```

Comment:
As I said above, contriving the examples for the sake of simplicity to the point they're no longer useful is probably not a good idea. We should strive for a compromise between usefulness and approachability. Unfortunately I'm not familiar with this tool in order to provide more useful (specific) feedback :/